xblockutils/resources.py

```diff
@@ -85,3 +88,14 @@ def load_scenarios_from_path(self, relative_scenario_dir, include_identifier=Fal
                     scenarios.append((identifier, title, scenario))
 
         return scenarios
+
+class MakoResourceLoader(ResourceLoader):
```

Comment:
Apologies @smarnach, I missed this comment before. I have now added ResourceLoader.render_mako_template and ResourceLoader.render_django_template and deprecated ResourceLoader.render_template.